src/core_plugins/tile_map/public/markers/scaled_circles.js

```diff
@@ -11,8 +11,8 @@ export class ScaledCirclesMarkers extends EventEmitter {
     this._geohashGeoJson = featureCollection;
     this._zoom = targetZoom;
 
-    this._valueFormatter = options.valueFormatter;
-    this._tooltipFormatter = options.tooltipFormatter;
+    this._valueFormatter = options.valueFormatter || ((x) => {x});
+    this._tooltipFormatter = options.tooltipFormatter || ((x) => {x});
```

Comment:
added unity functions so we don't need to mock ES-data for these properties in the unit tests. I'm working on https://github.com/elastic/kibana/pull/17263, which aims to simplify data-handling so we can use raw json to render the map visualizations.